bigbluebutton-html5/imports/ui/components/chat/component.jsx

```diff
@@ -66,6 +66,7 @@ const Chat = (props) => {
                 hideLabel
                 onClick={() => actions.handleClosePrivateChat(chatID)}
                 aria-label={intl.formatMessage(intlMessages.closeChatLabel, { 0: title })}
+                label=""
```

Comment:
label={intlMessages.closeChatLabel, { 0: title }}
perhaps? We have `hideLabel` so nothing will show aside from tooltip